pkg/operator/ceph/object/admin.go

```diff
@@ -106,7 +106,7 @@ const (
 	RGWAdminOpsUserSecretName = "rgw-admin-ops-user"
 	rgwAdminOpsUserAccessKey  = "accessKey"
 	rgwAdminOpsUserSecretKey  = "secretKey"
-	rgwAdminOpsUserCaps       = "buckets=*;users=*;usage=read;metadata=read;zone=read"
+	rgwAdminOpsUserCaps       = "buckets=*;users=read,write;usage=read;metadata=read;zone=read"
```

Comment:
Is `users=*` not equivalent to `users=read,write`?

Reply:
I changed to read only

Reply:
Is `write` needed for OBC to create users?